src/_posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md

```diff
@@ -1,53 +1,89 @@
 ---
-title: Getting started with Metabase on Scalingo
-modified_at: 2019-12-20 10:00:00
+title: Getting Started With Metabase on Scalingo
+modified_at: 2023-09-11 16:00:00
 tags: tutorial metabase
 index: 13
 ---
 
-Metabase is an open source tool to query your company data without writing SQL
+Metabase is an open source business intelligence tool that allows you to build
+dashboards and data visualization from your company data, without writing SQL
 queries.
 
 This tutorial will show you how to deploy a Metabase instance on Scalingo in
 under 5 minutes.
 
-## Metabase Deployment
+## Deploying Metabase
+
+### Planning your Deployment
+
+- Metabase requires its own database to store its configuration and some metadata.
+  We usually advise to use a [PostgreSQL Starter/Business 512 addon](https://scalingo.com/databases/postgresql)
+  for this purpose.
+
+- We also strongly advise against directly plugging Metabase to your production
```

Comment:
I'm not really agreeing on this.

* If database is small (< few GB), a read-only user is really good enough (performance and security wise)
* If database is larger, then a copy is required to prevent impacting the production database performance since BI queries may be costful.

Reply:
Ok :+1: 
Would it be possible to have a better estimation for "< few GB"? :smiley:

Reply:
Like hmmm ~5GB

Reply:
I finally chose another, more generic, wording.
Would that suit?